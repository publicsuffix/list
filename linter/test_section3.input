// test:
// - two PRIVATE sections

// ===BEGIN PRIVATE DOMAINS===

example.com

// ===END PRIVATE DOMAINS===

// ===BEGIN PRIVATE DOMAINS===
// ===END PRIVATE DOMAINS===
