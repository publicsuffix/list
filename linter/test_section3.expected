11: warning: No ICANN section found
11: warning: 2 PRIVATE sections found
